docs/content/preview/yugabyte-platform/create-deployments/create-universe-multi-zone-kubernetes.md

```diff
@@ -159,3 +159,47 @@ You can create a connection to a node as follows:
 ## Connect to the universe
 
 For information on how to connect to the universe from the Kubernetes cluster, as well as remotely, see [Connect YugabyteDB clusters](../../../deploy/kubernetes/clients/#connect-tls-secured-yugabytedb-cluster-deployed-by-helm-charts).
+
+### Create common TServer service for zones
+
+By default each zone gets its own TServer service which can be used to connect to the universe. Optionally, you can create a common service across these zones as follows. 
```

Comment:
If you create the common service, do the individual ones remain? What is the benefit?

Reply:
Mentioned that it is an additional service. 

About the benefits, I'm not sure how to write those here:
When you have a single service, your clients can connect to it and don't have to worry about individual TServers from different zones. If some zone's TServer(s) are down the request will be served by other zones TServer as the service takes care of load balancing. 

For now I have modified the sentence to say we create an highly available service with these steps.